### DISCUSSION TITLE
2006/02/25/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%9A%84%E5%8F%91%E5%B1%95%E3%80%81%E7%8E%B0%E7%8A%B6%E4%B8%8E%E5%B1%95%E6%9C%9B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#74] [Announcements]
# 2006/02/25/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%9A%84%E5%8F%91%E5%B1%95%E3%80%81%E7%8E%B0%E7%8A%B6%E4%B8%8E%E5%B1%95%E6%9C%9B

生物信息学的发展、现状与展望 (草稿)

https://blog.qizhen.xyz/2006/02/25/%E7%94%9F%E7%89%A9%E4%BF%A1%E6%81%AF%E5%AD%A6%E7%9A%84%E5%8F%91%E5%B1%95%E3%80%81%E7%8E%B0%E7%8A%B6%E4%B8%8E%E5%B1%95%E6%9C%9B

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Kuroda
2006年05月11日 12:17 编辑
http://crazypulsar.bokee.com/  我们连接吧  这是我的站站

回复
Kuroda
2006年05月11日 12:13 编辑
很喜欢你的站站~~~~~~~~~~~~~

回复
Tian
2006年03月16日 20:32 编辑
侬现在老揭滚哦！跟着你LP学了很多吧，惭愧惭愧，我现在对生物还是一窍不通

回复
